DOCS-6606-4.1-Intro-Mule-4-update-kt (#1096)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/intro-overview.adoc

= Introduction to Mule 4 for Mule 3 Users
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> If you are a user that already knows Mule 3 and recently migrated to Mule 4, review the following introduction overview to learn the main changes for this version. +

Reviewer: ```suggestion
If you are a user that already knows Mule 3 and recently migrated to Mule 4, review the following introduction to learn the main changes in Mule 4. +
```

> If you are an experienced Mule 4 user, jump directly to our xref:index.adoc[Mule Runtime] documentation instead.

Reviewer: ```suggestion
If you are an experienced Mule 4 user, go directly to our xref:index.adoc[Mule Runtime] documentation instead.
```

> Mule 4 simplifies the expression language and reduces management complexity so that you can speed up the on-ramping process and deliver applications faster than in Mule 3.

Reviewer: ```suggestion
Mule 4 simplifies the expression language and reduces management complexity so that you can ramp up and deliver applications faster than in Mule 3.
```

* xref:intro-error-handlers.adoc[Error Handling] - easier and more powerful error handling with a new Try scope.
* xref:intro-dataweave2.adoc[DataWeave] - DataWeave includes minor changes to simplify the syntax and make it easier to learn.
* xref:intro-studio.adoc[Studio 7] - features a simplified palette, improved Maven integration, and many other usability improvements.

> * xref:intro-engine.adoc[Runtime engine] - the internal execution engine has been updated with a new self-tuning and non-blocking reactive engine. This allows for better performance and scalability out-of-the-box.

Reviewer: I suggest using definition list format for this type of list. See style ref.
